portality/api/v1/data_objects/application.py

```diff
@@ -203,28 +233,17 @@ def __init__(self, raw=None):
                             "required" : ["count", "url"]
                         }
                     }
-                },
-
-                "admin" : {
-                    "lists" : {
-                        "contact" : {"contains" : "object"}
-                    },
-                    "required" : ["contact"],
-
-                    "structs" : {
-                        "contact": {
-                            "fields" : {
-                                "email" : {"coerce" : "unicode"},
-                                "name" : {"coerce" : "unicode"}
-                            },
-                            "required" : ["name", "email"]
-                        }
-                    }
                 }
             }
         }
 
-        super(IncomingApplication, self).__init__(raw, struct=struct, construct_silent_prune=True, expose_data=True)
+        mycoerce = deepcopy(self.DEFAULT_COERCE)
+        mycoerce["persistent_identifier_scheme"] = dataobj.string_canonicalise(["None", "DOI", "Handles", "ARK"], allow_fail=True)
```

Comment:
Would it be a good idea to get these from the form choices and string them together in a list dynamically? Rather than specifying explicitly here.